### PR TITLE
Blog: OT spring boot, describe screenshots

### PR DESCRIPTION
**Pull request by pavolloffay**
@jpkrohling @objectiser could you please review?

### CODE REVIEW

**Review by kevinearls — APPROVED** [review]
Looks good!

File: src/main/jbake/content/blog/2017/06/9/opentracing-spring-boot.adoc [hunk 1]
```diff
@@ -126,6 +129,10 @@ ifdef::env-github[]
 image::../../../../../assets/img/blog/2017/boot-zipkin-traces.png[caption="Figure 1: ", title="Zipkin showing reported traces."]
 endif::[]
 
+This screenshot also show traces for invocation of `/chaining` endpoint. In this case it shows only two spans because Zipkin
+uses so called shared span model which means that client and server invocation of `/hello` uses the same span.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/06/9/opentracing-spring-boot.adoc]: "uses a _shared span model_ which...."

File: src/main/jbake/content/blog/2017/06/9/opentracing-spring-boot.adoc [hunk 2]
```diff
@@ -126,6 +129,10 @@ ifdef::env-github[]
...
+This is a great example that different OpenTracing providers might model and show things differently.
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/06/9/opentracing-spring-boot.adoc]: "that shows how different"

**Review by objectiser — APPROVED** [review]
A couple of minor suggestions.

**Comment by pavolloffay** [comment]
@objectiser @kevinearls @pilhuhn thanks!

**Pull request merged**